File: Beecrowd/Beginner/1000 - 1099/1071 - Sum of Consecutive Odd Numbers I/Python/1071_SumofConsecutiveOddNumbersI.py
number1 = int(input())
number2 = int(input())
sum = 0

if number1 > number2:
    number1, number2 = number2, number1

for i in range(number1 + 1, number2):
    if i % 2 != 0:
        sum += i

print(sum)

File: Beecrowd/Beginner/1000 - 1099/1071 - Sum of Consecutive Odd Numbers I/C/1071_SumofConsecutiveOddNumbersI.c
#include <stdio.h>

int main() {
    int number1, number2, sum = 0;
    scanf("%d %d", &number1, &number2);

    if (number1 > number2) {
        int temp = number1;
        number1 = number2;
        number2 = temp;
    }

    for (int i = number1 + 1; i < number2; i++) {
        if (i % 2 != 0) {
            sum += i;
        }
    }

    printf("%d\n", sum);
    
    return 0;
}
